popSimManu.tex

```diff
@@ -208,22 +211,28 @@ \section*{Introduction}
 choosing pre-implemented models from a community-maintained catalog.
 The \stdpopsim catalog currently contains three organisms: humans,
 \emph{Drosophila~melanogaster}, and \emph{Arabidopsis~thaliana}. For each
-organism, the catalog contains details on the physical organization (e.g., chromosome structure)
-of its genome, one or more genetic maps, default population-level parameters (mutation rate,
-generation time) and one or more published demographic histories. Through
+organism, the catalog contains information on our current understanding of
+ the physical organization (e.g., chromosome structure)
+of its genome, one or more inferred genetic maps, default population-level parameters (mutation rate and generation time estimates) 
+and one or more published demographic models. These available models
+and parameters are meant to represent the current standards in the field, however our 
+intention is that this resource will evolve as new results become available. Through
 either a command line interface or a simple Python API, users can specify which
 organism, genetic map, chromosome, and demographic history they are interested in simulating, and the
 simulation output from their chosen model is returned.
 In this way, \stdpopsim will lower the barrier to high-quality simulation for exploratory analyses,
 enable rigorous evaluation of population genetic software,
 and contribute to increased reliability of population genetic inferences.
 
+
 The \stdpopsim library has been developed by the PopSim Consortium using a
 distributed open source model, with strong procedures in place
 to continue its growth and maintain its quality.
-Importantly, we have rigorous quality control methods to ensure implemented models are accurate
+Importantly, we have rigorous quality control methods to ensure that models
+are correctly implemented with respect to their original publication 
```

Comment:
maybe "correctly implement the models as described in their original publication"?